importAllData

```diff
@@ -4,4 +4,7 @@ set -u
 # Refresh source data, produce a data report and load the import mapping
 ./museum/importObjects -rpm
 # Run the import
-./museum/importObjects -i
+./museum/importObjects
+./museum/importSubjects -rm
```

Comment:
Inconsistent commenting (maybe just have a single comment at the top that describes the structure, i.e. "for each script, first the source data is refreshed, a data report is produced and the import mapping is loaded; and then in a second call, the import is run" (or something like that)).

Also, is there supposed to be a `-p` here?